<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QWidget" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{border-image:url(:/src/src/background.png)}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(0, 255, 255,0);border-radius:37;border-image:url(:/src/src/color.jpg)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:rgb(0, 0, 0); background:rgba(255, 255, 255, 0);border-image:url()}</string>
   </property>
   <property name="text">
    <string>253</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_sign">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:rgb(0, 0, 0); background:rgba(255, 255, 255, 0);border-image:url()}</string>
   </property>
   <property name="text">
    <string>好无聊呀~~</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_private">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{border:0dpx;border-image:url(:/src/src/private.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_public">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>61</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{border-image:url(:/src/src/public.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>381</width>
     <height>711</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(255, 255, 255, 0);border-image:url()}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>101</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgb(234, 193, 90);border-image:url()}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgba(234, 193, 90, 100);border-image:url()}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{background:rgb(234, 193, 90);border-image:url()}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
